torch/__init__.py

```diff
@@ -33,61 +34,44 @@
 # Load the extension module
 ################################################################################
 
-# Loading the extension with RTLD_GLOBAL option allows to not link extension
-# modules against the _C shared object. Their missing THP symbols will be
-# automatically filled by the dynamic loader.
-import os as _dl_flags
-
-# if we have numpy, it *must* be imported before the call to setdlopenflags()
-# or there is risk that later c modules will segfault when importing numpy
-try:
-    import numpy as _np
-except ImportError:
-    pass
-
 if platform.system() == 'Windows':
-    # first get nvToolsExt PATH
-    def get_nvToolsExt_path():
-        NVTOOLEXT_HOME = _dl_flags.getenv('NVTOOLSEXT_PATH', 'C:\\Program Files\\NVIDIA Corporation\\NvToolsExt')
+    NVTOOLSEXT_PATH = os.getenv('NVTOOLSEXT_PATH', 'C:\\Program Files\\NVIDIA Corporation\\NvToolsExt')
 
-        if _dl_flags.path.exists(NVTOOLEXT_HOME):
-            return _dl_flags.path.join(NVTOOLEXT_HOME, 'bin', 'x64')
-        else:
-            return ''
+    if os.path.exists(NVTOOLSEXT_PATH):
+        nvtoolsext_lib_path = os.path.join(NVTOOLSEXT_PATH, 'bin', 'x64')
+    else:
+        nvtoolsext_lib_path = ''
```

Comment:
A tip that maybe not directly related to this PR: We missed the path of CUDA here. To make it compatible with Python 3.8, we have to do that.

Reply:
I'm happy to fix this, but I'm not exactly sure what the suggestion is here. What's the other CUDA path we missed?

Reply:
Do we have a variable that records the version of the cuda used during build? If yes, then the answer is quite simple. For example, for CUDA 9.2, just add the following path.
```python
cuda_path = os.path.join(os.environ.get('CUDA_PATH_V9_2', r'C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.2'), 'bin')
```